### DISCUSSION TITLE
no `PyDateTime` in `types` ？

### DISCUSSION THREAD

**Discussion by sun-rs** [PyO3/pyo3#2389] [Questions] [answered]
With `pyo3 = { version = "0.16.5", features = ["hashbrown", "abi3-py37", "extension-module", "multiple-pymethods"] }`
When I put 'use pyo3::types::PyDateTime;' in my code file, it got an error:

`unresolved import pyo3::types::PyDateTime
no PyDateTime in types`

But `use pyo3::prelude::*; use pyo3::types::PyType;` works well

How to import PyDateTime?

**Comment by davidhewitt** [comment, marked as answer]
`PyDateTime` is not available when using `abi3` features - you'll have to remove the `abi3-py37` feature above.
